src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -127,15 +129,15 @@ public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguratio
                     +gvmPath.toString());
         }
         ProcessBuilder linkBuilder = new ProcessBuilder("gcc");
-        Path linux = gvmPath.resolve(appName);
+        Path linuxPath = gvmPath.resolve(appName);
 
         linkBuilder.command().add("-o");
-        linkBuilder.command().add(paths.getAppPath().toString() + "/" + appName);
-        linkBuilder.command().add(linux.toString() + "/launcher.o");
-        linkBuilder.command().add(linux.toString() + "/thread.o");
+        linkBuilder.command().add(paths.getAppPath().resolve(appName).toString());
+        linkBuilder.command().add(linuxPath.resolve("launcher.o").toString());
+        linkBuilder.command().add(linuxPath.resolve("thread.o").toString());
         linkBuilder.command().add(objectFile.toString());
         linkBuilder.command().add("-L" + projectConfiguration.getJavaStaticLibsPath());
-        linkBuilder.command().add("-L"+projectConfiguration.getGraalPath()+"/lib/svm/clibraries/"+target.getOsArch2());// darwin-amd64");
+        linkBuilder.command().add("-L"+ Path.of(projectConfiguration.getGraalPath(),"lib","svm","clibraries",target.getOsArch2()));// darwin-amd64");
```

Comment:
Add space after commas.

Reply:
Fixed. Please look it over as there are few more small tweaks.